Makefile

```diff
@@ -5,8 +5,12 @@ VERSION = $(shell python -c "print(__import__('didadata').__version__)")
 clean:
 	rm -fr docs/_build build/ dist/
 
-tests:
-	py.test --cov
+format-python-code:
+	isort -rc .
+	black .
+
+tests: clean
```

Comment:
Why clean before tests?

Reply:
If I run `py.test` locally, tests raise some error. After some more tries I randomly run `make tests` and all tests passed. I don´t know why 🤔 So I thought it would be better to call `clean` before running tests

Reply:
What kind of error?

Reply:
Oha I don´t know anymore and I can´t reproduce it.

Reply:
So should I remove the clean command in tests?

Reply:
If I run `make tests` without including the `clean` command many tests fail:
```
$ make tests
py.test
/Users/benb/.virtualenvs/django-didadata/lib/python3.8/site-packages/_pytest/terminal.py:287: PytestDeprecationWarning: TerminalReporter.writer attribute is deprecated, use TerminalReporter._tw instead at your own risk.
See https://docs.pytest.org/en/latest/deprecations.html#terminalreporter-writer for more information.
  warnings.warn(
=========================================================================================== test session starts ============================================================================================
platform darwin -- Python 3.8.1, pytest-5.4.1, py-1.8.1, pluggy-0.13.1 -- /Users/benb/.virtualenvs/django-didadata/bin/python
cachedir: .pytest_cache
django: settings: tests.settings (from ini)
rootdir: /Users/benb/Sites/django-didadata, inifile: setup.cfg, testpaths: didadata/, tests/
plugins: black-0.3.8, isort-0.3.1, flake8-1.0.4, cov-2.8.1, django-3.8.0
collected 66 items

tests/test_admin.py::TestMetricAdmin::test_list_display Creating test database for alias 'default'...
PASSED
tests/test_admin.py::TestRecordAdmin::test_list_display PASSED
tests/test_admin.py::TestRecordAdmin::test_list_filter PASSED
tests/test_admin.py::TestRecordAdmin::test_date_hierarchy PASSED
tests/test_api.py::TestRecordApi::test_invalid_metric PASSED
tests/test_api.py::TestRecordApi::test_invalid_value PASSED
tests/test_api.py::TestRecordApi::test_created PASSED
tests/test_api.py::TestRecordApi::test_created_with_timestamp PASSED
tests/test_api.py::TestRecordApi::test_created_with_timestamp_invalid_format PASSED
tests/test_models.py::TestMetricModel::test_repr PASSED
tests/test_models.py::TestRecordModel::test_repr PASSED
didadata/__init__.py::FLAKE8 SKIPPED
didadata/__init__.py::ISORT SKIPPED
didadata/__init__.py::BLACK SKIPPED
didadata/admin.py::FLAKE8 SKIPPED
didadata/admin.py::ISORT SKIPPED
didadata/admin.py::BLACK SKIPPED
didadata/api.py::FLAKE8 SKIPPED
didadata/api.py::ISORT SKIPPED
didadata/api.py::BLACK SKIPPED
didadata/apps.py::FLAKE8 SKIPPED
didadata/apps.py::ISORT SKIPPED
didadata/apps.py::BLACK SKIPPED
didadata/exceptions.py::FLAKE8 SKIPPED
didadata/exceptions.py::ISORT SKIPPED
didadata/exceptions.py::BLACK SKIPPED
didadata/filters.py::FLAKE8 SKIPPED
didadata/filters.py::ISORT SKIPPED
didadata/filters.py::BLACK SKIPPED
didadata/models.py::FLAKE8 SKIPPED
didadata/models.py::ISORT SKIPPED
didadata/models.py::BLACK SKIPPED
didadata/serializers.py::FLAKE8 SKIPPED
didadata/serializers.py::ISORT SKIPPED
didadata/serializers.py::BLACK SKIPPED
didadata/urls.py::FLAKE8 SKIPPED
didadata/urls.py::ISORT SKIPPED
didadata/urls.py::BLACK SKIPPED
didadata/viewsets.py::FLAKE8 SKIPPED
didadata/viewsets.py::ISORT SKIPPED
didadata/viewsets.py::BLACK SKIPPED
didadata/migrations/__init__.py::BLACK SKIPPED
didadata/migrations/0001_initial.py::BLACK SKIPPED
didadata/migrations/0002_auto_20151201_1939.py::BLACK SKIPPED
didadata/migrations/0003_auto_20160311_2321.py::BLACK SKIPPED
tests/__init__.py::FLAKE8 SKIPPED
tests/__init__.py::ISORT SKIPPED
tests/__init__.py::BLACK SKIPPED
tests/settings.py::FLAKE8 PASSED
tests/settings.py::ISORT PASSED
tests/settings.py::BLACK PASSED
tests/test_admin.py::FLAKE8 SKIPPED
tests/test_admin.py::ISORT SKIPPED
tests/test_admin.py::BLACK SKIPPED
tests/test_api.py::FLAKE8 SKIPPED
tests/test_api.py::ISORT SKIPPED
tests/test_api.py::BLACK SKIPPED
tests/test_models.py::FLAKE8 SKIPPED
tests/test_models.py::ISORT SKIPPED
tests/test_models.py::BLACK SKIPPED
tests/factories/__init__.py::FLAKE8 SKIPPED
tests/factories/__init__.py::ISORT SKIPPED
tests/factories/__init__.py::BLACK SKIPPED
tests/factories/metrics.py::FLAKE8 SKIPPED
tests/factories/metrics.py::ISORT SKIPPED
tests/factories/metrics.py::BLACK SKIPPEDDestroying test database for alias 'default'...
```

Reply:
Aaaaahhhhhh I got it! pytest creates a `.pytest_cache` dir. This directory must be removed before run tests.

Reply:
Now I think I got good solution for this problem. Have a look at setup.cfg:
addopts = -vs --cache-clear --tb=short --black --isort --flake8  --nomigrations

I added --cache-clear

Reply:
Ok, after we discussed the `--cache-clear` argument I removed it ;)